### PR TITLE
Add the Crystal Linux Logo

### PR DESCRIPTION
**Pull request by AloneER0**
I added the Crystal Linux logo, but i don't know how to change the color of OS: Host: etc i hope you get what i mean oh and the syntax may be a f up but yeah i tried 😅

### CODE REVIEW

**Comment by AloneER0** [comment]
image
This is what i mean

File: src/logo/builtin.c [hunk 1]
```diff
@@ -522,6 +522,50 @@ static const FFlogo* getLogoCentOSSmall()
...
+"            dysssym\n"
+"          dysssym\n"
+<<<<<<< HEAD
+"        dysssym\n"
+
+
+    )              
+=======
+"        dysssym\n"            
+>>>>>>> 5f8a27faa53c1beba2131d4a1638119ac478dff9
```
Comment (LinusDierheimer) [inline src/logo/builtin.c:556]: Please remove this part from the pull request, this is not a patch

File: src/logo/builtin.c [hunk 2]
```diff
@@ -522,6 +522,50 @@ static const FFlogo* getLogoCentOSSmall()
...
+    
+    FF_LOGO_COLORS(
+        "35", //magenta
+        "30" //black
```
Comment (LinusDierheimer) [inline src/logo/builtin.c:560]: Why is black given here? Your logo template doesn't use a second color

**Review by LinusDierheimer — COMMENTED** [review]
Please add the logo name to the README.md too!

> I don't know how to change the color of OS: Host: etc

They have the color you set with FF_LOGO_COLOR_KEYS(...)

File: src/logo/builtin.c (hunk 1 above)
Comment (AloneER0) [reply src/logo/builtin.c:556]: How 😅

Comment (LinusDierheimer) [reply src/logo/builtin.c:556]: Remove it from you local code. Then:
```bash
git add .
git commit -m "Removed patch code"
git push
```

Comment (AloneER0) [reply src/logo/builtin.c:556]: Ohh that sorry i don't know how it got there :(

Comment (AloneER0) [reply src/logo/builtin.c:556]: let me fix it

**Comment by LinusDierheimer** [comment]
There are build errors, please resolve them.

**Comment by AloneER0** [comment]
On it.

**Comment by AloneER0** [comment]
Fixed it

**Comment by LinusDierheimer** [comment]
You need to push your changes

**Comment by AloneER0** [comment]
Give me a sec

**Comment by AloneER0** [comment]
Pushed check now

**Comment by LinusDierheimer** [comment]
Thanks!

**Pull request merged**